File: Unidad 5/c#/Programa_8_Unidad_5.cs
using System;

namespace Programa_8_Unidad_5 // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int numero_Dados = 2;
            int resultado = Lanzar_Dados(numero_Dados);

            Console.WriteLine("Resultado del lanzamiento: " + resultado);
        }

        static int Lanzar_Dados(int numero_Dados)
        {
            Random random = new Random();
            int resultado = 0;

            for (int i = 0; i < numero_Dados; i++)
            {
                int valor_Dado = random.Next(1, 7);
                Console.WriteLine($"Dado {i + 1}: {valor_Dado}");

                resultado += valor_Dado;
            }

            return resultado;
        }
    }
}

File: Unidad 5/Java/Programa_8_Unidad_5.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa_8_unidad_5;
import java.util.Random;
/**
 *
 * @author USUARIO LENOVO
 */
public class Programa_8_Unidad_5 {

    public static void main(String[] args) {
        int numero_Dados = 2; 
        int resultado = lanzar_Dados(numero_Dados);

        System.out.println("Resultado del lanzamiento: " + resultado);
    }

    public static int lanzar_Dados(int numero_Dados) {
        Random random = new Random();
        int resultado = 0;

        for (int i = 0; i < numero_Dados; i++) {

            int valor_Dado = random.nextInt(6) + 1;
            System.out.println("Dado " + (i + 1) + ": " + valor_Dado);

            resultado += valor_Dado;
        }

        return resultado;
    }
}
